setup.cfg

```diff
@@ -44,6 +44,9 @@ install_requires =
 python_requires = >=3.6.1
 
 [options.packages.find]
+include =
+  allennlp_plugins.*
+  allennlp_server
```

Comment:
This is not strictly needed actually, without it the built package will be the same. However, maybe it's good to have because it adds any path a priori:

https://github.com/pypa/setuptools/blob/73376585065bbf28395c71fe15137c19a712d4f3/setuptools/__init__.py#L116-L119

As opposed to only those with an `__init__.py` file in it:

https://github.com/pypa/setuptools/blob/73376585065bbf28395c71fe15137c19a712d4f3/setuptools/__init__.py#L102-L105

Reply:
Albeit maybe it's better to include things that shouldn't be there than not having stuff in the package, so maybe I should remove it. I open to comments on this.

Reply:
But you removed `find` and changed it to `find_namespace`.  Does this still do anything?

Reply:
Or maybe I'm just not parsing this file correctly...

Reply:
> But you removed `find` and changed it to `find_namespace`. Does this still do anything?

Yes. If you look at `setuptools` code, the only difference is in the method to accept a directory as included. `find` (which is actually `find_packages`) only accepts those that have an `__init__.py` file in it. `find_namespace` (`find_namespace_packages`) accepts everything. Even though is not necessary now, I whitelisted because in the future some extra folder could get in unintentionally.

If you mean if the `find`/`find_namespace` is necessary when whitelisting, my understanding is yes because the folder `allennlp_plugins` won't pass the condition of having an `__init__.py` file even if whitelisted.

Reply:
I was just wondering about `options.packages.find` as the header here (a few lines above this), while you switched from `find` to `find_namespace` above.  If you say this still is fine, I'll trust you.  I would rather not worry about learning how all of this works myself =).

Reply:
Yes, it's fine.